packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -17,7 +17,7 @@
 import * as Ajv from 'ajv';
 import { inject, injectable, interfaces, named, postConstruct } from 'inversify';
 import { ContributionProvider, bindContributionProvider, escapeRegExpCharacters, Emitter, Event } from '../../common';
-import { PreferenceScope } from './preference-service';
+import { PreferenceScope } from './preference-scope';
```

Comment:
It's a breaking change. Would it be possible to reexport `PreferenceScope` from `preference-service` as before?

Reply:
Yes unfortunately I noticed it would be a breaking change :(

The idea was to remove the circular dependency between the three files since they all use `PreferenceScope` and it is present in **preference-service**.  

Is there a way to fix the issue without resulting in a breaking change?

<div align='center'>

![screenshot from 2019-03-06 07-46-08](https://user-images.githubusercontent.com/40359487/53882368-29ecf800-3fe4-11e9-8ee9-1a6c2ab63f7e.png)


</div>

Reply:
add `export { PreferenceScope };` to `preference-service`

Reply:
thank you @akosyakov, it's been fixed :)